docs/proposals/1374-mc-inference-gateways/README.md

```diff
@@ -0,0 +1,165 @@
+# Multi-Cluster Inference Gateways
+
+Author(s): @robscott, @bexxmodd
+
+## Proposal Status
+
+ ***Draft***
+
+## Summary
+
+Inference Gateways aim to provide efficient routing to LLM workloads running in Kubernetes. In practice, an Inference Gateway is a Gateway that conforms to the [Gateway API Inference Extension](https://gateway-api-inference-extension.sigs.k8s.io/). This Gateway supports a new type of backend - InferencePool. When routing to an [InferencePool](https://gateway-api-inference-extension.sigs.k8s.io/api-types/inferencepool/), the Gateway calls out to an “Endpoint Picker” referenced by the InferencePool to get instructions on which specific endpoint within the pool it should reference.
+
+![Inference Architecture](images/gw-epp-ip.png)
+
+### Why Multi-Cluster?
+
+Until now, Inference Gateways have been focused exclusively on routing to a single cluster. Unfortunately, the resources needed to run LLM workloads continue to be scarce, and the desired capacity is rarely available within a single cluster. To address this, we propose expanding InferencePool to support multi-cluster routing.
+
+### Goals
+
+Enable Inference Gateways to route to backends in multiple clusters
+Follow a pattern that is familiar to users of Multi-Cluster Services (MCS) and/or Gateways
+
+### Non-Goals
+
+Be overly prescriptive about implementation details - this should focus on the resulting UX and leave significant flexibility in how it is achieved
```

Comment:
Each should be a bullet and end with a period.

Reply:
> My working theory here is that GPU costs and desire for optimal performance will likely be more important for most users than any topology-aware routing initially

@robscott agreed, and I think the correllary to this is that specific GPU/network needs would likely be addressed best through DRA scheduling onto appropriate nodes rather than as a routing concern